doc/development-context-aware-applications/how-to-update-and-query-context-information.md

```diff
@@ -66,101 +81,201 @@ it in the following way:
       }
     } 
 
-You can also obtain the values of all attributes of the "LeBistro"
+You can also obtain the values of all attributes of the "Elizalde"
 restaurant in a single shot:
 
-    GET <cb_host>:<cb_port>/v1/contextEntities/type/Restaurant/id/LeBistro
-
-    //getting a JSON response such as the following one:
 
+    GET <cb_host>:<cb_port>/v1/contextEntities/type/Restaurant/id/Elizalde
     {
-      "contextElement": {
-        "attributes": [
-        {
-          "name": "name",
-          "type": "string",
-          "value": "Le Bistro"
-        },
-        {
-          "name": "average_scoring",
-          "type": "float",
-          "value": "4.2"
-        },
+      "contextElement":
         {
-          "name": "location",
-          "type": "location",
-          "value": "40.419697, -3.691281",
-          "metadatas": [
-          {
-            "name": "location",
-            "type": "string",
-            "value": "WGS84"
-          }
-        },
-        {
-          "name": "postal_address",
-          "type": "string",
-          "value": "Calle Alcala 57, 28028 Madrid, Spain"
+          "type": "Restaurant",
+          "isPattern": "false",
+          "id": "Elizalde",
+          "attributes":
+          [
+            {
+              "name": "address",
+              "type": "",
```

Comment:
strange that the type is the empty string. If so, probably it is a problem of the feeding Tour Guide App. Please, could you double check with Bitergia Team?